.github/workflows/release-e2e-workflow-template.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           tar -xzf opensearch-${{ env.VERSION }}-linux-x64.tar.gz
           cd opensearch-${{ env.VERSION }}/
           ./opensearch-tar-install.sh &
-          timeout 900 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
+          timeout 3600 bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' -u admin:admin -k https://localhost:9200)" != "200" ]]; do sleep 5; done'
```

Comment:
why increase timeout? I think this will affect all test cases.

Reply:
Previously just tried to increase the timeout to see if E2E check can pass. But did not work. I changed back to 900.